ex2/ex2.c

```diff
@@ -1,14 +1,63 @@
-// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory 
-// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor 
+// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory
+// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor
 // returned by `fopen()`? What happens when they are written to the file concurrently?
 
 #include <stdio.h>
 #include <unistd.h>
 #include <stdlib.h>
 
-int main(int argc, char* argv[])
+int main(int argc, char *argv[])
 {
-    // Your code here 
-    
+    char text[100];
+    FILE *fp;
+
+    fp = fopen("text.txt", "r");
```

Comment:
The intention of this exercise was to have you write to the file, but I think I forgot to remove the contents from the `text.txt` file, so this isn't that surprising that you'd interpret the instructions this way.

Reply:
oh my.. whoops